fdbserver/SkipList.cpp

```diff
@@ -56,64 +48,19 @@ static inline int skfastrand() {
 
 void setAffinity(int proc);
 
-class SlowConflictSet {
-public:
-	bool is_conflict( const VectorRef<KeyRangeRef>& readRanges, Version read_snapshot );
-	void add( const VectorRef<KeyRangeRef>& clearRanges, const VectorRef<KeyValueRef>& setValues, Version now );
-	void clear( Version now );
-
-private:
-	KeyRangeMap<Version> age;
-};
-
-bool SlowConflictSet::is_conflict( const VectorRef<KeyRangeRef>& readRanges, Version read_snapshot ) {
-	for(auto range = readRanges.begin(); range != readRanges.end(); ++range) {
-		auto intersecting = age.intersectingRanges( *range );
-		for(auto it = intersecting.begin(); it != intersecting.end(); ++it)
-			if ( it.value() > read_snapshot )
-				return true;
-	}
-	return false;
-}
-
-void SlowConflictSet::clear( Version now ) {
-	age.insert(allKeys, now);
-}
-
-void SlowConflictSet::add( const VectorRef<KeyRangeRef>& clearRanges, const VectorRef<KeyValueRef>& setValues, Version now ) {
-	for(auto c = clearRanges.begin(); c != clearRanges.end(); ++c)
-		age.insert( *c, now );
-	for(auto s = setValues.begin(); s != setValues.end(); ++s)
-		age.insert( s->key, now );
-}
-
-
-PerfDoubleCounter 
-	g_buildTest("Build", skc),
-	g_add("Add", skc),
-	g_add_sort("A.Sort", skc),
-	g_detectConflicts("Detect", skc),
-	g_sort("D.Sort", skc),
-	g_combine("D.Combine", skc),
-	g_checkRead("D.CheckRead", skc),
-	g_checkBatch("D.CheckIntraBatch", skc),
-	g_merge("D.MergeWrite", skc),
-	g_merge_launch("D.Merge.Launch", skc),
-	g_merge_fork("D.Merge.Fork", skc),
-	g_merge_start_var("D.Merge.StartVariance", skc),
-	g_merge_end_var("D.Merge.EndVariance", skc),
-	g_merge_run_var("D.Merge.RunVariance", skc),
-	g_merge_run_shortest("D.Merge.ShortestRun", skc),
-	g_merge_run_longest("D.Merge.LongestRun", skc),
-	g_merge_run_total("D.Merge.TotalRun", skc),
-	g_merge_join("D.Merge.Join", skc),
-	g_removeBefore("D.RemoveBefore", skc)
-	;
-
-static force_inline int compare( const StringRef& a, const StringRef& b ) {
-	int c = memcmp( a.begin(), b.begin(), min(a.size(), b.size()) );
-	if (c<0) return -1;
-	if (c>0) return +1;
+PerfDoubleCounter g_buildTest("Build", skc), g_add("Add", skc), g_add_sort("A.Sort", skc),
```

Comment:
If you would prefer the previous formatting for this, I could force all of these onto their own line with comments.

Reply:
I think clang-format is doing the right thing here.  Separating this out into many statements instead would be preferable, and then clang-format will also do the right thing for that.